src/documentation/text/01_Preface/requirements.tex

```diff
@@ -0,0 +1,66 @@
+In diesem Kapitel werden die zu erfüllenden Eigenschaften und Leistungen an das System gesammelt. Dadurch werden strukturierte und konkrete Anforderungen an das Gesamtsystem bereitgestellt.
+Nachfolgend wird zwischen funktionalen und nicht-funktionalen Anforderungen unterschieden.
+Da die Anforderungen ausschließlich der Aufgabenstellung entstammen, findet keine Gewichtung der einzelnen Anforderungen hinsichtlich der Umsetzungsrelevanz statt.
+
+\subsubsection{Funktionale Anforderungen}\label{sec:functional_requirements}
+
+Funktionale Anforderungen beschreiben die gewünschten spezifischen Funktionalitäten.\ifootcite[37]{grande2011}
+Tabelle X beinhaltet die funktionalen Anforderungen an das Gesamtsystem.
+
+\newcounter{tablenumber}
+\newcommand\inctablenumber{\stepcounter{tablenumber}}
+\newcommand\tablenumber{\arabic{tablenumber}}
+\newcounter{rownumber}
+\newcommand\incrownumber{\stepcounter{rownumber}}
+\newcommand\rownumber{\arabic{rownumber}}
+\setcounter{tablenumber}{1}
+\setcounter{rownumber}{1}
+\begin{dhbwtable}{caption={Allgemeine Anforderungen},label=tab:requirements,float=H}
+    \begin{tabularx}{\textwidth}{l | X}
+        \toprule
+        \textbf{ID} & \textbf{Beschreibung}  \\\midrule
+        \tablenumber .\rownumber & Kunde und Verwalter sollen über Clients auf die Verwaltungskomponente zugreifen können. \\\midrule\incrownumber
+        \tablenumber .\rownumber & Die Anzahl der verfügbaren Tickets einer Veranstaltung soll begrenzt sein. \\\midrule\incrownumber
+        \tablenumber .\rownumber & Die Anzahl der verkauften Tickets einer Veranstaltung pro Kunde soll begrenzt sein. \\\midrule\incrownumber
```

Comment:
Die Anzahl der zu kaufenden Tickets einer Veranstaltung pro Kunde soll begrenzt sein. ?

Reply:
![image](https://user-images.githubusercontent.com/47746780/90310030-0541c300-deee-11ea-858a-146138419b8a.png)

Verstehe ich so

Reply:
In der Aufgabenstellung steht auf Folie 2:
> Für jede Veranstaltung gibt es eine vorgegebene Anzahl von Tickets.
Die Anzahl der verkauften Tickets pro Kunde ist begrenzt.

Habt ich das falsch aufgenommen?

Ihr hattet in eurem Google Docs Dokument geschrieben:
> Durch das System soll die Anzahl an Tickets pro Veranstaltung für einen Kunden begrenzt werden können.

Reply:
ohman sorry hatte die Aufgabenstellung nicht mehr im Kopf fand es nur beim lesen komisch